modules/rpc/yarpc.go

```diff
@@ -39,12 +39,12 @@ import (
 // YarpcModule is an implementation of a core module using YARPC
 type YarpcModule struct {
 	modules.ModuleBase
-	rpc          yarpc.Dispatcher
-	register     registerServiceFunc
-	config       yarpcConfig
-	log          ulog.Log
-	stateMu      sync.RWMutex
-	interceptors []transport.Interceptor
+	rpc                yarpc.Dispatcher
+	register           registerServiceFunc
+	config             yarpcConfig
+	log                ulog.Log
+	stateMu            sync.RWMutex
+	inboundMiddlewares []transport.UnaryInboundMiddleware
```

Comment:
We've renamed interceptors and filters to inboundMiddleware and outboundMiddleware. It's a bit more verbose, but people we're getting confused with the previous terminology.